src/constants.js

```diff
@@ -1,2 +1,3 @@
 export const DEFAULT_QUERY_LIMIT = 50;
 export const MAXIMUM_QUERY_LIMIT = 500;
+export const AWS_PROFILE = 'texvet';
```

Comment:
This AWS_PROFILE is not defined in the CircleCI environment where this will run. In fact, I don't know what this is intended to do. This seeder doesn't do anything in AWS. All it does is update the MongoDB collections that are outside of AWS. This should be removed along with every line that uses it.